lib/ansible/module_utils/basic.py

```diff
@@ -1305,6 +1305,9 @@ def log(self, msg, log_args=None):
                 # TODO: surrogateescape is a danger here on Py3
                 journal_msg = remove_values(msg, self.no_log_values)
 
+            if self._target_log_info:
+                journal_msg = remove_values(self._target_log_info, self.no_log_values) + journal_msg
```

Comment:
Isn't this going to crash the target log message into the front of `journal_msg` (no space or other delimiter)? Also, should probably only call remove_values on whatever the final message is, rather than calling it repeatedly. Since this is 2.17, we can also probably dump most if not all the bytes support (or at worst, just `to_text` all the args once on the way in and call it good.

Reply:
fixed the things introduced here, leaving the bytes handling fix for subsequent PR fater DT